packages/taquito/src/operations/types.ts

```diff
@@ -425,6 +426,24 @@ export interface RPCIncreasePaidStorageOperation {
   destination: string;
 }
 
+/**
+ * @description Ballot operation params
+ */
+export interface BallotParams {
+  source?: string;
+  period: number;
+  proposal: string;
+  ballot: OperationContentsBallotEnum;
```

Comment:
The name `OperationContentsBallotEnum` doesn't seem 100% appropriate in my opinion. Should we create another enum specific for the params?

Reply:
I think that already existed from the RPC package. But we can definitely improve it.

perhaps `BallotEnum` is a bit more concise?